app/qml/PreviewPanel.qml

```diff
@@ -92,6 +102,12 @@ Item {
                 anchors.top: header.bottom
                 anchors.bottom: parent.bottom
 
+                MouseArea {
+                    anchors.fill: parent
+                    onClicked: contentClicked()
+                    z: content.z + 1
+                }
```

Comment:
why do we need this here when there is already a mouse area for the full size (around line 20) ?